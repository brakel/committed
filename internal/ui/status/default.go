package status

func defaultModifiers() []Modifier {
	return []Modifier{
		{Modifier: noModifier, Align: alignRight},
		{Modifier: shiftModifier, Label: "Shift", Align: alignRight},
		{Modifier: altModifier, Label: "Alt", Align: alignLeft},
		{Modifier: controlModifier, Label: "Ctrl", Align: alignLeft},
	}
}

func defaultShortcuts() []Shortcut {
	return []Shortcut{
		{Modifier: controlModifier, Key: "c", Label: "Cancel"},
		{Modifier: altModifier, Key: "enter", Label: "Commit"},
		{Modifier: altModifier, Key: "s", Label: "Sign-off"},
		{Modifier: altModifier, Key: "/", Label: "Help"},
	}
}
