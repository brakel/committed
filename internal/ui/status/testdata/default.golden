 Alt + <enter> Commit <a> Amend <l> Load <s> Sign-off
Ctrl +     <c> Cancel <h> Help
